packages/sdk/src/types/runtime/rigidbody.ts

```diff
@@ -23,6 +23,7 @@ export interface RigidBodyLike {
     detectCollisions: boolean;
     collisionDetectionMode: CollisionDetectionMode;
     useGravity: boolean;
+    isKinematic: boolean;

```

Comment:
This whole interface needs documentation, but could you start here? Use Material for reference: [https://github.com/microsoft/mixed-reality-extension-sdk/blob/red/packages/sdk/src/types/runtime/assets/material.ts](https://github.com/microsoft/mixed-reality-extension-sdk/blob/red/packages/sdk/src/types/runtime/assets/material.ts)

Reply:
Sure.  I am fine with beginning the documentation updates.

Reply:
Thanks for adding documentation.